data/assets/scene/solarsystem/sun/sun.asset

```diff
@@ -18,7 +18,7 @@ local Sun = {
   },
   ApproachFactor = 15.0,
   GUI = {
-    Name = "Sun",
+    Name = "Solar Surface",
```

Comment:
```suggestion
    Name = "Sun",
```

It is the sun, the texture that we put on by default happens to be the photosphere, but other users can put other textures on it

Reply:
I've tossed this around in my brain... I see your point. We don't say "Earth Surface" or "Mars Surface."